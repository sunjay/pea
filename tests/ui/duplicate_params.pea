fn foo(value, value) {
    // variables are allowed to shadow params, but params are not allowed to shadow each other
    let value = 3;
    println!(value);
}
