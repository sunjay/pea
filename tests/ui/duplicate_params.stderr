error: identifier `value` is bound more than once in this parameter list
[tests/ui/duplicate_params.pea:1:15-1:19] error: used as parameter more than once

error: aborting due to 1 previous error

